File: recaudacion/2.Codigo/backend/RecaudacionApiLiquidacion/Domain/TipoDocumento.cs
using System;

namespace RecaudacionApiLiquidacion.Domain
{
    public class TipoDocumento
    {
        public int TipoDocumentoId { get; set; }
        public string Nombre { get; set; }
        public string Abreviatura { get; set; }
        public bool Estado { get; set; }
    }
}

File: recaudacion/2.Codigo/frontend/src/app/core/models/tipodocumento.ts
export class TipoDocumento {
  tipoDocumentoId: number;
  nombre: string;
  abreviatura: string;
  estado: boolean;
  usuarioCreador: string;
  usuarioModificador: string;
  index:number;
}

export class TipoDocumentoEstado {
  tipoDocumentoId: number;
  estado: boolean;
  usuarioModificador: string;
}
export class TipoDocumentoFilter {
  pageNumber: number;
  pageSize: number;
  sortColumn: string;
  sortOrder: string;
  nombre: string;
  estado: boolean;
}
